eclipse-scout-core/src/calendar/Calendar.ts

```diff
@@ -845,19 +963,20 @@ export class Calendar extends Widget implements CalendarModel {
       gridPaddingX = this.$grid.innerWidth() - this.$grid.width(),
       gridW = containerW - gridPaddingX;
 
-    // show or hide year
-    $('.calendar-toggle-year', this.$commands).select(this._showYearPanel);
-    if (this._showYearPanel) {
-      this.yearPanel.$container.data('new-width', this.calendarToggleYearWidth);
+    // show or hide calendar sidebar
+    if (this.showCalendarSidebar) {
+      this.calendarSidebar.$container.data('new-width', this.calendarToggleYearWidth);
       gridW -= this.calendarToggleYearWidth;
       containerW -= this.calendarToggleYearWidth;
     } else {
-      this.yearPanel.$container.data('new-width', 0);
+      this.calendarSidebar.$container.data('new-width', 0);
     }
 
+    this.calendarSidebar.setCalendarsPanelExpanded(this.showCalendarsPanel);
```

Comment:
I think, it makes sense to be able to modify the property, event when the sidebar is not expanded. Because maybe you want, that the Tree is visible directly when you expand the sidebar. Otherwise, you have to click twice...

Reply:
I agree => resolved